### DISCUSSION TITLE
I use Macrodroid to disable/enable hotspot

### DISCUSSION THREAD

**Discussion by RetoHB** [tesla-android/issue-tracker#170] [Show and tell]
When I am parked I want my car to disconnect from Tesla Android, so I can access the car over Internet. So I installed Macrodroid on Tesla Android with these two macros:

![Screenshot_20221123-103445](https://user-images.githubusercontent.com/9882700/203514219-f99e1c7e-3579-453b-8960-c5f3ef71ea34.jpg)

When I disconnect my USB thethering phone the hotspot is disabled, when I reconnect my phone the hotspot comes back on. Seems to work so far.

**Comment by adamChrystie** [comment]
This seems useful if you do not have a LTE USB modem attached to the PI...or if you want to use Tesla data instead of the LTE Modem's SIM card due to monthly data limits. Thanks for sharing.

**Reply by RetoHB** [reply]
I have no LTE modem, this is why.
